parcels/kernels/EOSseawaterproperties.py

```diff
@@ -0,0 +1,348 @@
+"""Collection of pre-built eos sea water property kernels"""
+import math
+
+__all__ = ['pressure_from_latdepth', 'adtg', 'ptemp_from_temp', 'temp_from_ptemp']
```

Comment:
All other built-in kernels are Capitalised. So also Capitalise the functions here?